client/src/utils/formatPrice.js

```diff
@@ -1 +1,5 @@
-console.log();
+export default function formatPrice(price) {
+  let number = price.replace(/[^0-9]/g, '');
+  const convertPrice = number.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
+  return '₩ ' + convertPrice;
```

Comment:
template literal 사용하셔도 좋을것 같네요~~

```suggestion
  return `₩ ${convertPrice}`;
```

Reply:
네! template litreal이 더 깔끔하니 보기 좋네요!